examples/README.md

```diff
@@ -0,0 +1,93 @@
+# AWS Encryption SDK Examples
+
+Here you can find some examples that show you
+how to use the AWS Encryption SDK.
+We demonstrate how to use the high-level APIs
+as well as how to set up some common configuration patterns.
+
+## APIs
+
+The AWS Encryption SDK provides two high-level APIS:
+one-shot APIs that process the entire operation in memory
+and streaming APIs.
+
+You can find examples that demonstrate these APIs
+in the [`examples/src/`](./src) directory root.
+
+## Configuration
+
+In order to use the library APIs,
+you must provide some configuration that defines
+how you want to protect your data keys.
+
+### Keyrings
+
+Keyrings are the most common way for you to configure that AWS Encryption SDK.
+These let you define how you want the AWS Encryption SDK to protect your data keys.
+You can find these examples in [`examples/src/keyring`](./src/keyring).
+
+### Cryptographic Materials Managers
+
+Keyrings define how you want to protect your data keys,
+but there is more going on here than just data keys.
+
+Cryptographic materials managers give you higher-level controls
```

Comment:
higher-level or lower-level? I think of the CMM as lower-level.

Reply:
Higher-level than keyrings.